assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -112,6 +114,9 @@ const CartLineItemRow = ( { lineItem = {} } ) => {
 					disabled={ isPendingDelete }
 				/>
 				<ProductLowStockBadge lowStockRemaining={ lowStockRemaining } />
+				{ showBackorderNotification && (
+					<ProductBackorderNotification />
+				) }
```

Comment:
I'm thinking maybe we need to combine the badge logic into a single component. I _think_ we only ever want to show one badge at a time - they aren't independent. 

In my testing I got `low stock` and `on backorder` showing by bumping up the quantity of a low-stock, allows-backorder product:

<img width="1172" alt="Screen Shot 2020-07-13 at 10 41 13 AM" src="https://user-images.githubusercontent.com/4167300/87258334-e3bc7880-c4f6-11ea-901e-c5128dff784e.png">

I think if the product is on backorder, we show the backorder badge only – it supercedes the low-stock badge. So `low stock` means shopper 'has the last one' and should order fast, `backorder` means that one or more items is on backorder, so shipping might be a while off.

Reply:
Good catch. In 4c26498c0f063763e0592ccb698cf08a2280eea1 I made it so the stock badge doesn't appear if the backorder badge is shown.